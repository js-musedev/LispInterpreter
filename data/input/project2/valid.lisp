(DEFUN F34 (ER) (N 34 (ER 23424)))
123
(3 5 (XYZ) 7)
(NIL 5 ( ) (( )) 7 (( ) 9 ( )) )
(DEFUN F23 (X) (PLUS X 12 55))