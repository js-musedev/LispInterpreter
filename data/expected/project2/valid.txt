Error! Empty input not allowed!